README.md

```diff
@@ -1,81 +1,45 @@
-    
-## Questions
-Before rasing a question, please check it hasn't been alerady answered in
-1. [Our FAQ section](https://docs.mongock.io/v5/FAQ/index.html)
-2. [Github Q&A section](https://github.com/mongock/mongock/discussions/categories/q-a)
-3. [Stackoverflow](https://stackoverflow.com/search?q=mongock)
-4. [Gitter channel](https://gitter.im/mongock/community)
-    
-If you cannot find the answer you are looking for, feel free to raise a new question in our [Gitter channel](https://gitter.im/mongock/community) or [Q&A section](https://github.com/mongock/mongock/discussions/categories/q-a)
-
-## Issues
-
-To raise a new issue, please take a look to [this guide](https://docs.mongock.io/v5/contributing/index.html)
-
-## Changelog
-To see our changelog, please take a look to  our [github releases][github_releases]
-
-## Older versions support
-With new major releases we stop enhancing or adding new features to older version(like version 3 or 4). However, we'll keep providing support and bug fixes for a long period. We are working on an official support plan.
-
-
-## Code of conduct
-Please read the [code of conduct][codeOfConduct] for details on our code of conduct.
-
-## LICENSE
-Mongock project is licensed under the [Apache License Version 2.0][apacheLicense]. See the [LICENSE][mongockLicense] file for details
-    
-## Contributing
-:tada: First off, thanks for taking the time to contribute! :tada:
-    
-If you would like to contribute to the Mongock project, visit our [official documentation](https://docs.mongock.io/v5/contributing/index.html)
-    
-    
-<a href="https://github.com/mongock/mongock/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=mongock/mongock" />
-</a>
-
-    
-
-[codeOfConduct]: ./CODE_OF_CONDUCT.md
-[mongockLicense]: ./LICENSE.md
-
-[apacheLicense]: http://www.apache.org/licenses/LICENSE-2.0.html
-[documentation_link]: https://docs.mongock.io/
-[github_releases]: https://github.com/mongock/mongock/releases
+# ⚠️ Mongock is Deprecated
+
+**Status:** This project is no longer maintained (as of 2025-08-01).  
+No new features will be added. Critical security fixes may be considered on a best-effort basis only.
+
+> For future-proof projects, we recommend transitioning to [Flamingock](https://github.com/flamingock/flamingock-project).
+
+---
+
+## Meet Flamingock 🚀
+
+We are the original authors of Mongock. Over the past couple of years, we've been heads‑down building **[Flamingock](https://github.com/flamingock/flamingock-project)** — a platform inspired by what worked in Mongock, but with a much broader scope: **auditable, deterministic evolution across your entire system**. (A managed cloud offering is also in the works.)
+
+- Robust support for MongoDB and NoSQL migrations, covering everything previously offered by Mongock
+- Extends beyond databases to orchestrate changes across message queues, contracts, APIs, configs, feature flags, and more
```

Comment:
- Extends beyond databases to orchestrate changes across any component type (message queues, APIs, databases, feature flags, etc.)